onnx/cpp2py_export.cc

```diff
@@ -193,7 +193,7 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
       [](const py::bytes& bytes, const std::vector<std::string>& names) {
         ModelProto proto{};
         ParseProtoFromPyBytes(proto, bytes);
-        auto const result = optimization::Optimize(proto, names);
+        auto const result = optimization::Optimize(std::move(proto), names);
```

Comment:
I don't think we need this. `Optimize` is taking `const ModelProto&` which won't incur a copy.

Reply:
@yinghai It's not for avoiding copy when passing to optimizer, it's for freeing the `proto` once the `Optimize` call is done (previously it's won't be freed until the end of this pybind11 interface function.

Reply:
I see. This will only work with newer version of protobuf, but it shoud be good.

Reply:
since you already turn this into a rvalue reference, why don't you change the Optimize() API to take a rvalue reference.